sdk_contrib/aws_v1_support/build.gradle

```diff
@@ -0,0 +1,19 @@
+description = 'OpenTelemetry SDK AWS SDK v1.x Support'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    api project(':opentelemetry-api'),
+            project(':opentelemetry-sdk')
+
+    implementation libraries.guava,
+            libraries.disruptor
```

Comment:
Why this dependency?

Reply:
Just as a follow-on, we have this dependency in the SDK right now, as well, and it should probably be removed, until we actually want to change the implementation of the SpanProcessors to use it (if we ever want to).

Reply:
do we? I don't see that. https://github.com/open-telemetry/opentelemetry-java/blob/master/sdk/build.gradle

Reply:
https://github.com/open-telemetry/opentelemetry-java/blob/master/build.gradle#L120

Reply:
not the SDK, sorry..the root of the project.

Reply:
That's where we define the dependency but it needs to be added to the api/implementation to become a real dependency for the artifact. We use it in the async_span_processor artifact.

Reply:
that's fair. just seems weird to declare it here, when it's only used in one sub-module.